### DISCUSSION TITLE
Is the english dictionary US or GB

### DISCUSSION THREAD

**Discussion by NathanTech7713** [barrust/pyspellchecker#119] [Q&A] [answered]
Subject basically sums it up but:
1. Is it US english or GB english and
2. 2. If it is one or the other, how hard of a time would it be to separate those into two separate dictionaries?
3.

Thanks

**Comment by barrust** [comment, marked as answer]
The English dictionary is built off of [this data set](http://opus.nlpl.eu/download.php?f=OpenSubtitles/v2018/mono/OpenSubtitles.raw.en.gz). I do not believe it distinguishes between the different flavors, but I could be wrong. 

As for how to separate them (assuming they are the same) is unknown. Not sure if there is an easy method unless the sources were separated out.

**Reply by GreenReaper** [reply]
It doesn't appear to support 'colour' but it does allow 'color' so I'm guessing there is at least a bias towards the US. (The data set itself appears to be gone now?)

**Reply by barrust** [reply]
looks like they changed the URLS: https://object.pouta.csc.fi/OPUS-OpenSubtitles/v2018/mono/en.txt.gz

**Comment by davebulaval** [comment]
To improve dictionary quality, we could consider using recent huge dataset like [GigaWord](https://huggingface.co/datasets/gigaword) (~4 M documents) or the insanely huge [The Pile](https://huggingface.co/datasets/EleutherAI/pile) (825 GO) dataset.

The latter is more diverse (medical document, emails, etc.).

I have a script that can run both dataset and output a JSON dict.
